### PR TITLE
[zh] sync/blog/_posts/2022-08-16-PSP-historical-context/index.md

### PR DESCRIPTION
**Pull request by liangyongzhenya**

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: blog
 title: "PodSecurityPolicy：历史背景"
 date: 2022-08-23T15:00:00-0800
 slug: podsecuritypolicy-the-historical-context
+evergreen: true
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:6]:
```suggestion
evergreen: true
```
Please add the same content behind line 13

Comment (liangyongzhenya) [reply content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md:6]: thx～～

**Comment by windsonsea** [comment]
Thanks.

/lgtm

**Pull request merged**